packages/@lwc/engine-server/src/__tests__/fixtures/getter-class-list/modules/x/getter-class-list/getter-class-list.js

```diff
@@ -9,5 +9,6 @@ export default class GetterClassList extends LightningElement {
         classList.remove('b');
 
         expect(this.getAttribute('class')).toBe('a c d-e');
+        expect(() => this.classList.length).not.toThrow();
```

Comment:
```suggestion
        expect(this.classList.length).toBe(3);
```
We know what it should be, why not check that directly?

Reply:
I had that originally. Original SSR implementation returns `undefined`. When we retire `@lwc/engine-server`, we may want to revisit things like this.